tests/test_solver.py

```diff
@@ -121,40 +124,66 @@ def _submit_and_check(self, solver, linear, quad, **kwargs):
         return results
 
 @unittest.skipUnless(config, "No live server configuration available.")
-class Utilities(_QueryTest):
+class Utilities(unittest.TestCase):
     """Test solver utility methods."""
 
-    def test_qpu_access_time_estimate(self):
-        with Client(**config) as client:
-            solver = client.get_solver(qpu=True)
-
-            runtime_q1000 = solver.estimate_qpu_access_time(num_qubits=1000)
+    def test_qpu_access_time_estimate_mock(self):
```

Comment:
This test should not be guarded by `skipUnless` from above. We want to run as many tests we can if offline.

Reply:
Thanks, was a leftover from when all the tests were together